code/game/gamemodes/meteor/meteor.dm

```diff
@@ -24,10 +24,10 @@
 
 
 /datum/game_mode/meteor/proc/sendmeteors()
-	var/waveduration = world.time + rand(0,1000) + text2num("[wave]000")
-	var/waitduration = rand(3000,9000)
+	var/waveduration = world.time + rand(0,1000) + text2num("[wave]000" / 2)

```

Comment:
@ZomgPonies, you divided a string by 2 and then passed the result as a string. might be why the mode doesn't work